binary/builder/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG BUILDER_IMAGE=fission/builder:latest
+ARG BUILDER_IMAGE=harshthakur/builder:test
```

Comment:
We will have to change those references?

Reply:
Yes! We'll have to make and push those images to fission's account, I'll rename it and then we can merge it